draft-ietf-quic-tls.md

```diff
@@ -715,6 +715,11 @@ method described in Section 7.3 of {{!TLS13}}), with the labels "quic key" and
 respectively).  Using these labels provides key separation between QUIC and TLS,
 see {{key-diversity}}.
 
+The HKDF-Expand-Label function defined in TLS 1.3 MUST be used even when a
+different TLS version is negotiated, or - for Initial packets - if the TLS
+versions offered do not include TLS 1.3.  This ensures that the selection of TLS
+version is independent of the process used to derive QUIC keys.
```

Comment:
It might be useful to describe the nuance here  -- that Initial packets are expected to be decrypted before a TLS version is negotiated, and therefore there needs to be prior agreement on the HKDF-Expand-Label function used for it.